ui/dist/index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>9Volt UI</title>
+  </head>
+  <body>
+    <div id="root" />
+    <script src="/dist/bundle.js" type="text/javascript"></script>
```

Comment:
The entire `html` of the project is wrapped up in this line. It's using the `/dist` handler in the `api.go` to serve the `bundle.js`. `bundle.js` is built by [Webpack](https://webpack.github.io/) which allows you to collapse code and resources down to a reasonable collection of artifacts. *Webpack* builds what we put into the `statik.go` file.